Mu2eG4/src/Mu2eG4PrimaryGeneratorAction.cc

```diff
@@ -256,6 +256,11 @@ namespace mu2e {
       }
 
     }
+    else if (pdgId == PDGCode::geantino) {
+      // AE: for some reason GetParticleTable()->FindParticle(0) does not find geantino
+      // but GetParticleTable()->FindParticle("geantino") does
+      pDef = G4ParticleTable::GetParticleTable()->FindParticle("geantino");
```

Comment:
I looked at the G4 source code.  It explicitly rejects a PDG code of zero - I guess it knows that the PDG does not define that code.    I agree that you have found the best solution.